webapp/components/Table/Header/Paginator/Paginator.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import { useI18n } from '@webapp/store/system'
+import { getLink } from '@webapp/components/Table/tableLink'
+
+const Paginator = (props) => {
+  const { offset, limit, count } = props
+  const currentPage = offset / limit + 1
+  const totalPage = Math.ceil(count / limit)
+
+  const i18n = useI18n()
+
+  return (
+    <div className="table__paginator">
+      <Link className="btn btn-transparent" aria-disabled={count < limit || currentPage === 1} to={getLink(0)}>
```

Comment:
`currentPage === 1`

should we extract this to `isFirstPage`?

Reply:
ok done